src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/ProxyRoutesSpec.scala

```diff
@@ -47,208 +47,212 @@ class ProxyRoutesSpec extends FlatSpec with BeforeAndAfterAll with BeforeAndAfte
     proxyService.googleTokenCache.invalidateAll()
   }
 
-  "ProxyRoutes" should "listen on /notebooks/{project}/{name}/..." in {
-    Get(s"/notebooks/$googleProject/$clusterName").addHeader(Cookie(tokenCookie)) ~> leoRoutes.route ~> check {
-      handled shouldBe true
-      status shouldEqual StatusCodes.OK
-      validateCors()
+  val pathPrefixes = Set("notebooks", "proxy")
+
+  for(prefix <- pathPrefixes) {
```

Comment:
this file is all the same, it's just nested inside of a loop to test the new routes and the deprecated ones